rostime/include/ros/duration.h

```diff
@@ -64,6 +64,16 @@ namespace ros
 ROSTIME_DECL void normalizeSecNSecSigned(int64_t& sec, int64_t& nsec);
 ROSTIME_DECL void normalizeSecNSecSigned(int32_t& sec, int32_t& nsec);
 
+// Arithmetic oprations of unsigned 32/64 bit numbers with these signed constants will convert these constants to
+// unsigned numbers according to the C standard. 16 bit unsigned numbers will get implicitly converted to int32_t for
+// the computation.
+constexpr int32_t NSecInUSec = 1000l;  //!< Number of nanoseconds in a microsecond.
```

Comment:
I find these name particularly confusing. I assume you want to express that there are e.g. 1000 ns in a us. But when using these constants the resulting expression is really hard to read / interpret.

I would find a name like `USecToNSec` much more intuitive. So in order to convert a "usec" value to "nsec" it would be multiplied with that constant. That makes expressions much easier to read imo: `nsec_value = usec_value * USecToNSec`.

Reply:
Renamed as suggested (and made names uppercase).